From FDWM commit 5224add fix(dwm): read the pointer position in resizemouse for tiled windows too (#18)

diff --git a/dwm.c b/dwm.c
index 60ca0b5..2c21e86 100644
--- a/dwm.c
+++ b/dwm.c
@@ -1379,10 +1379,8 @@ resizemouse(const Arg *arg)
 		None, cursor[CurResize]->cursor, CurrentTime) != GrabSuccess)
 		return;
 
-	if (c->isfloating) {
-		if(!getrootptr(&x, &y))
-			return;
-	}
+	if (!getrootptr(&x, &y))
+		return;
 
 	do {
 		XMaskEvent(dpy, MOUSEMASK|ExposureMask|SubstructureRedirectMask, &ev);
