From FDWM commit c65c01c chore(suckless): remove transient.c and the unused dist targets (#D5)

diff --git a/Makefile b/Makefile
index 86f9ef2..6fdea38 100644
--- a/Makefile
+++ b/Makefile
@@ -23,16 +23,7 @@ stest: stest.o
 	$(CC) -o $@ stest.o $(LDFLAGS)
 
 clean:
-	rm -f dmenu stest $(OBJ) dmenu-$(VERSION).tar.gz *.rej *.orig config.def.h README
-
-dist: clean
-	mkdir -p dmenu-$(VERSION)
-	cp LICENSE Makefile arg.h config.h config.mk dmenu.1\
-		drw.h util.h dmenu_path dmenu_run stest.1 $(SRC)\
-		dmenu-$(VERSION)
-	tar -cf dmenu-$(VERSION).tar dmenu-$(VERSION)
-	gzip dmenu-$(VERSION).tar
-	rm -rf dmenu-$(VERSION)
+	rm -f dmenu stest $(OBJ) *.rej *.orig config.def.h README
 
 install: all
 	mkdir -p $(DESTDIR)$(PREFIX)/bin
@@ -55,4 +46,4 @@ uninstall:
 		$(DESTDIR)$(MANPREFIX)/man1/dmenu.1\
 		$(DESTDIR)$(MANPREFIX)/man1/stest.1
 
-.PHONY: all clean dist install uninstall
+.PHONY: all clean install uninstall
